packages/vsce/README.md

```diff
@@ -124,71 +124,71 @@ Right-click a CICS profile and select `Manage Profile` to show profile options.
 
-**Tip:** The default 500 count can be modified via the `Record Count Increment` property in Settings (UI).
+**Tip:** The default 500 count can be modified via the `Record Count Increment` property in Settings (UI). **[What does "(UI)" mean here? I don't see it in the screenshot.]**
 
 <p align="center">
 <img src="/packages/vsce/docs/images/record-count-increment.png" alt="Zowe CICS Explorer Record Count Increment in Setting UI" width="700px"/>
 </p>
 
-### Show Attributes
+### Show attributes
 
 Right-click and use the pop-up menu against a program to list the available actions that can be performed. For every resource, including a CICS region, `Show Attributes` opens a viewer listing all attributes and their values. The attributes page has a filter box at the top that lets you search for attributes matching the criteria.
 
 <p align="center">
 <img src="/packages/vsce/docs/images/show-attributes.gif" alt="Zowe CICS Explorer Filter" width="700px"/>
 </p>
 
-### Enable and Disable
+### Enable and disable
 
-Right-click against a program, local transaction or local file to open up the pop-up menu and click `Disable [CICS resource]` to disable the resource. When a resource is already disabled the first option becomes `Enable [CICS resource]` to allow its enablement state to be toggled. A disabled resource is identified by a `(Disabled)` text next to its name.
+Right-click against a program, local transaction, or local file to open the pop-up menu and select `Disable [CICS resource]` to disable the resource. When a resource is already disabled the first option becomes `Enable [CICS resource]` to allow its enablement state to be toggled. A disabled resource is identified by a `(Disabled)` label next to its name.
 
 <p align="center">
 <img src="/packages/vsce/docs/images/disable-enable.gif" alt="Zowe CICS Explorer Filter" width="700px"/>
 </p>
 
 ### New Copy and Phase In
 
-Use `New Copy` <img src="/packages/vsce/docs/images/program-newcopy-action.png" width="16px"/> and `Phase In` <img src="/packages/vsce/docs/images/program-phasein-action.png" width="16px"/> actions against a CICS program to get the CICS region to load a fresh of the selected program into memory. This could be after you've edited a COBOL program source and successfully compiled it into a load library and now want to test your change.
+Use `New Copy` <img src="/packages/vsce/docs/images/program-newcopy-action.png" width="16px"/> and `Phase In` <img src="/packages/vsce/docs/images/program-phasein-action.png" width="16px"/> actions against a CICS program to get the CICS region to load a fresh of **[is there a word missing here? "a fresh of" what?]** the selected program into memory. This could be after you have edited a COBOL program source and successfully compiled it into a load library so that you can test your change.
```

Comment:
@davenice - please respond

Reply:
Thanks - I've updated the wording to be inspired by the CICS doc - https://www.ibm.com/docs/en/cics-ts/5.6?topic=sc-set-program

I hadn't spotted this flagrant typo! Good catch :-)